rfs_test/TEST_accountservice_schema.py

```diff
@@ -18,283 +18,473 @@
+
+        # 201 is the only status when a request has created a new resource successfully
+
+        if status == 201:
+            assertion_status = log.FAIL
+            log.assertion_log('line', "Assertion Failed")
+            return assertion_status
 
-# NOT NECESSARY  
+    except:
+        assertion_status = log.WARN
+        log.assertion_log('line', "~ \'AccountsService\MaxPasswordLength' not found in the payload from GET %s" % (
+            '/redfish/v1/AccountService'))
+        return assertion_status
 
-###################################################################################################
-# Name: Assertion_ACCO111(self, log) :Account Service                                            
-# Assertion text: 
-# The Actions property shall contain the available actions for this resource.
-###################################################################################################
+    log.assertion_log(assertion_status, None)
+    log.assertion_log('line', "Assertion Passes")
+    return assertion_status
 
 
-## end Assertion_ACCO111
+# end Assertion_ACCO104
 
-# NOT NECESSARY  
 
 ###################################################################################################
-# Name: Assertion_ACCO112(self, log) :Account Service                                            
-# Assertion text: 
-# The Actions property shall contain the available actions for this resource.
+# Name: Assertion_ACCO105(self, log) :Account Service
+# Assertion text:
+# This property shall reference the threshold of failed login attempts at which point the user's
+# account is locked.  If set to 0, no lockout shall ever occur.
 ###################################################################################################
 
+def Assertion_ACCO105(self, log):
+
+    log.AssertionID = 'ACCO105'
+    assertion_status = log.PASS
+    log.assertion_log('BEGIN_ASSERTION', None)
+
+    if not clearLog(self):
+        assertion_status = log.WARN
+        log.assertion_log('line', "Could not clear the Service Log")
+        return assertion_status
+
+    relative_uris = self.relative_uris
+    authorization = 'on'
+    rq_headers = self.request_headers()
     
-## end Assertion_ACCO112
- 
-# NOT NECESSARY  
+    json_payload, headers, status = self.http_GET(
+        '/redfish/v1/AccountService', rq_headers, authorization)
+
+    try:
+        AccountLockoutThreshold = json_payload['AccountLockoutThreshold']
+
+        for i in range(0, AccountLockoutThreshold):
+            if not failAuthorization(self): 
+                assertion_status = log.WARN
+                log.assertion_log('line', "Could not fail the authorization")
+                return assertion_status 
+            try: 
+                authorization = 'on'
+                json_payload, headers, status = self.http_GET('/redfish/v1/Managers/MultiBladeBMC/LogServices/Log/Entries/', rq_headers, authorization)
```

Comment:
Would recommend trying to access /redfish/v1/AccountService with invalid credentials; the URI here is likely to not be present on a given system.

In addition, we don't need to test for a log entry being added to a given log service when an account is locked out.